src/imex.rs

```diff
@@ -258,7 +258,7 @@ async fn maybe_add_bcc_self_device_msg(context: &Context) -> Result<()> {
         msg.text = Some(
             "It seems you are using multiple devices with Delta Chat. Great!\n\n\
              If you also want to synchronize outgoing messages across all devices, \
-             go to the settings and enable \"Send copy to self\"."
+             go to the Settings → "Advanced" → enable \"Send copy to self\"."
```

Comment:
```suggestion
             go to \"Settings → Advanced\" and enable \"Send copy to self\"."
```

at least, this is how we describe click paths in the help. in the app, we use "Settings / Advanced", but i would prefer the "→" as well.